src/Sarif.Viewer.VisualStudio/ErrorListService.cs

```diff
@@ -163,9 +163,11 @@ private void WriteRunToErrorList(Run runLog)
                         HelpLink = rule?.HelpUri?.ToString()                        
                     };
 
-                    CaptureAnnotatedCodeLocationCollections(result.Stacks, AnnotatedCodeLocationKind.Stack, sarifError);
+                    IEnumerable<IEnumerable<AnnotatedCodeLocation>> stackLocations = CreateAnnotationsFromStacks(result.Stacks);
+
+                    CaptureAnnotatedCodeLocationCollections(stackLocations, AnnotatedCodeLocationKind.Stack, sarifError);
                     CaptureAnnotatedCodeLocationCollections(result.CodeFlows, AnnotatedCodeLocationKind.CodeFlow, sarifError);
-                     CaptureAnnotatedCodeLocations(result.RelatedLocations, AnnotatedCodeLocationKind.Stack, sarifError);
+                    CaptureAnnotatedCodeLocations(result.RelatedLocations, AnnotatedCodeLocationKind.Stack, sarifError);
```

Comment:
Ah yes, I see -- when we introduce the intermediate `codeFlow` object we've discussed, you'll need to have an overload of `CreateAnnotatedCodeLocationsCollections` that takes `result.CodeFlows`.